src/runtime/components/nuxt-img.ts

```diff
@@ -78,6 +78,7 @@ export default defineComponent({
       useHead({
         link: [{
           rel: 'preload',
+          nonce: props.nonce,
```

Comment:
```suggestion
          ...(props.nonce ? { nonce: props.nonce } : {}),
```

Reply:
I see you removed this quite quickly after my recent push in https://github.com/nuxt/image/pull/1012/commits/d1d5258b9711f8669b70ef94e5c1d32987e4d90b 

Locally `pnpm test:types` (on https://github.com/nuxt/image/pull/1012/commits/07b141e0d82b742ed81f908911af898a819bee48) runs without error (and without the need for the `ts-expect-error`).

Reply:
sorry, I was editing locally at the same time 🙈

the reason I prefer this is that the spread _does_ work without type errors, but it also removes type safety here as you could pass anything in, e.g. `{ bob: true }`.

Reply:
I see, makes sense! Thanks for all your hard work on nuxt btw :)